services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationGetStarted.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Col, Row, Typography } from "antd";
+
+export const MajorMineApplicationGetStarted = () => {
+  return (
+    <Row>
+      <Col style={{ marginLeft: "15%" }}>
+        <Typography.Title level={3}>
+          Getting Started with your Major Mine Submission
+        </Typography.Title>
+        <Typography.Paragraph>
+          Basic information about what a major mine submission is will go here
+        </Typography.Paragraph>
+        <Typography.Title level={4}>Basic Information</Typography.Title>
+        <Typography.Paragraph>Information what is needed for this section</Typography.Paragraph>
+        <Typography.Title level={4}>Upload Primary Document</Typography.Title>
+        <Typography.Paragraph>Information what is needed for this section</Typography.Paragraph>
+        <Typography.Title level={4}>Upload Supporting Documents</Typography.Title>
+        <Typography.Paragraph>Information what is needed for this section</Typography.Paragraph>
```

Comment:
Looking at the mocks this is missing the "Upload Spatial Documents" section and it's copy. Also do we have the paragraph copy ready yet or will that be addressed in another ticket?